internal/cmd/operator_uninstall_test.go

```diff
@@ -0,0 +1,223 @@
+package cmd_test
```

Comment:
I don't see any actual usage of the `internal/cmd` package in this test. It seems more appropriate to put this file (and ` internal/cmd/cmd_suite_test.go`) in `internal/action` instead.